spec/models/user_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe(User, type: :model) do
+  let(:user) { build(:user) }
+
+  context 'When user entered valid fields' do
+    it { expect(user).to be_valid }
+  end
+
+  context 'When user blank' do
+    it { expect(User.new).to_not be_valid }
+  end
+
+  describe 'name' do
```

Comment:
Have you read the difference between `context ` and `describe `?

Reply:
You cannot describe model fields only methods!

Create a general `context ` which says 'user validation' and then you can have specific `context`s or just `it` blocks with a clear description